### PR TITLE
Add ProductGrid

### PR DESCRIPTION
**Pull request by maximilian-dz**
Add ProductGrid - Responsive except on MD
Add products.js file that contains all product-data and props.
Add temp-images

### CODE REVIEW

File: src/components/ProductGrid.js [hunk 1]
```diff
@@ -1,7 +1,30 @@
...
 
 const ProductGrid = () => {
-  return <section></section>
+  return (
+    <section>
+      <div className="container">
+        <div className="shop-bg margin-bottom-50px"></div>
+        <div className="row">
+          {products &&
+            products.map((product, index) => {
```
Comment (scarsam) [inline src/components/ProductGrid.js:12]: Här kan du returera direkt istället för att skriva `return` 

`{products && products.map((product, index) => (
      <Product
       ...
))};

**Comment by scarsam** [comment]
@maximilian-dz Netlify failar också

**Comment by scarsam** [comment]
@maximilian-dz Från Netlify logs:
```
Error: icon (src/images/gatsby-icon.png) does not exist as defined in gatsby-c  onfig.js. Make sure the file exists relative to the root of the site.
```

File: src/components/ProductGrid.js (hunk 1 above)
Comment (maximilian-dz) [reply src/components/ProductGrid.js:12]: Haha vafan

**Pull request merged**